src/bin/sol-fbp-runner/inspector.c

```diff
@@ -201,6 +203,20 @@ inspector_show_packet(const struct sol_flow_packet *packet)
             fprintf(stdout, "<lat=%g|lon=%g|alt=%g>", v.lat, v.lon, v.alt);
             return;
         }
+    } else if (type == SOL_FLOW_PACKET_TYPE_TIMESTAMP) {
+        struct timespec v;
+        if (sol_flow_packet_get_timestamp(packet, &v) == 0) {
+            struct tm cur_time;
+            char buf[32];
+            tzset();
+            if (localtime_r(&v.tv_sec, &cur_time)) {
```

Comment:
I need review this more closely, but it looks that you should be using `gmtime_r()` instead of `localtime_r()`, as you are printing the value in the UTC timezone ('Z').

Am I missing something?